lib/ferrum/browser.rb

```diff
@@ -125,6 +125,9 @@ class Browser
     # @option options [Hash] :env
     #   Environment variables you'd like to pass through to the process.
     #
+    # @option options [Boolean] :use_default_context
```

Comment:
I don't know what the best name for this is... it's kinda confusing since `Contexts#default_context` is already a thing and the current behavior of that method is to create a new BrowserContext. But I felt like "default" should be the right thing because the effective change is to not pass a browserContextId when [creating a target](https://chromedevtools.github.io/devtools-protocol/tot/Target/#method-createTarget).

Perhaps a less confusing name would be `use_persistent_context`? Similar to how Playwright has `launchPersistentContext`.